### PR TITLE
dist: build linux/darwin arm64 targets

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
Adding the linux/darwin arm64 targets, we start building arm64 for the server and will start adding the targets for the plugins for the ones we prepackage in the build

related to: https://github.com/mattermost/mattermost-server/pull/18491

#### Ticket Link
related to https://mattermost.atlassian.net/browse/DOPS-633

### CODE REVIEW

File: server/manifest.go [hunk 1]
```diff
@@ -2,43 +2,10 @@
...
-`
-
-func init() {
-	manifest = model.ManifestFromJson(strings.NewReader(manifestStr))
+var manifest = struct {
+	ID      string
+	Version string
+}{
+	ID:      "com.mattermost.plugin-channel-export",
+	Version: "1.0.0",
```
Comment (hanzei) [inline server/manifest.go:10]: I don't have an objection to removing the JSON representation of the manifest, but I'm wondering why this changes was made.

**Review by hanzei — APPROVED** [review]

File: server/manifest.go (hunk 1 above)
Comment (cpanato) [reply server/manifest.go:10]: i've updated `build/manifest/main.go` to match the other plugins and then this happened 
I can revert that as well

Comment (hanzei) [reply server/manifest.go:10]: 1/5 that the previous implementation is better.

Comment (cpanato) [reply server/manifest.go:10]: give me a sec

Comment (cpanato) [reply server/manifest.go]: done @hanzei

File: server/manifest.go [hunk 2]
```diff
@@ -2,10 +2,46 @@
...
+  "settings_schema": {
+    "header": "",
+    "footer": "",
+    "settings": []
+  }
+}
+`
+
+func init() {
+	var manifest model.Manifest
```
Comment (hanzei) [inline server/manifest.go:45]: This line need to be removed. Otherwise `manifest` is present in the local scope.

Comment (cpanato) [reply server/manifest.go]: thanks for the catch!
